crates/ide/src/runnables.rs

```diff
@@ -483,7 +483,11 @@ fn has_runnable_doc_test(attrs: &hir::Attrs) -> bool {
         let mut in_code_block = false;
 
         for line in String::from(doc).lines() {
-            if let Some(header) = line.strip_prefix(RUSTDOC_FENCE) {
+            if let Some(header) = RUSTDOC_FENCES
+                .map(|fence| line.strip_prefix(fence))
+                .iter()
+                .fold(None as Option<&str>, |a, b| a.or(*b))
+            {
```

Comment:
```suggestion
        if let Some(header) = RUSTDOC_FENCES
            .into_iter()
            .find_map(|fence| line.strip_prefix(fence))
        {
```
No need for the odd folding dance here and elsewhere

Reply:
Thanks for letting me know it! I amended it. 🙏🏻